packages/fakta-beregning/src/components/avklareAktiviteter/avklareAktiviteterPanel.module.css

```
@@ -2,3 +2,15 @@
   font-size: 1.125rem;
   font-weight: 600;
 }
+
+.colNavn {
+  width: 40%;
+}
+
+.colRadio {
+  width: 5%;
+}
+
+.colPeriode {
+  width: 15%;
+}
```

man kan oppnå det samme med å erstatte denne på _<table>_

```suggestion
.table{
  table-layout:fixed
}
```